pom.xml

```diff
@@ -103,6 +103,12 @@
 
 	<dependencies>
 		<dependency>
+			<groupId>org.awaitility</groupId>
+			<artifactId>awaitility</artifactId>
+			<version>3.1.2</version>
+			<scope>test</scope>
+		</dependency>
+    <dependency>
```

Comment:
This indent is still wrong.
Can you, please, be sure that you use tabs for indents in your IDE?
Plus reformat the whole POM to ensure common style.

Thanks